docs/conf.py

```diff
@@ -274,4 +274,5 @@
     (_py_class_role, 'UUID'),
     (_py_class_role, 'WebsocketFrame'),
     (_py_class_role, 'Url'),
+    (_py_class_role, '_asyncio.Task'),
```

Comment:
@webknjaz This is kind of annoying, specially because there is no hint from the editor itself.  Until one run the CI, they won't discover such nitty things, resulting in slow iteration, discouragement from contributions.   E.g. if a person need to make 3-turns to just fix CI, it's simply discouraging.

Either we should look into some sort of Editor configuration (I already checked in my `.vscode`), or relax such warnings which are simply unnecessary.  Evaluating by weight, value that this adds is not worth it to slow down contributions and iterations.

PS: Problem is locally entire flow cannot be run, specially because of `pylint`.  There are other issues I observed e.g. stale content in `.tox`, `doc` where I had to delete these folders to get going.  I will list them down separately.

Reply:
This is because of the autogenerated API docs. You cannot do anything about this. These are generated from annotations that sometimes point at things that are not documented in the stdlib docs. You could disable turning warnings into errors entirely and allow committing broken docs for which you won't have any signals for whether everything is correct or not.

Reply:
But why do we have to add our own lib classes here e.g. `WebsocketFrame` and such

Reply:
It depends on the exact case. I can't guess a universal answer. For the cases where it's coming from auto-generated pages, it may be that I excluded a few modules because they were causing unsolvable problems. Another case would be when you manually try referring to this object but use a wrong/incomplete reference. Show me where it happens and I'll tell you why exactly.

Reply:
Oh, you're probably referring to the one I added originally. I was ignoring a lot of things because it was easier to get the initial setup merged this way. Those should be fixed. Most of the objects are documented and should be referred to via their full names. As in `package.subpackage.module.WebsocketFrame`. But if you just use `WebsocketFrame`, it's hard for Sphinx to guess what you mean. There's syntax for making sure that when you use the full reference, it'll still be rendered as the short one. It RST, it needs a tilde:
```rst
:class:`~package.subpackage.module.WebsocketFrame`
```
the above will be rendered as `WebsocketFrame` in HTML while referencing a specific documented object.
If you do
```rst
:class:`package.subpackage.module.WebsocketFrame`
```
it will be rendered as `package.subpackage.module.WebsocketFrame`.
You can also enter a custom anchor:
```rst
:class:`some thing <package.subpackage.module.WebsocketFrame>`
```
This one will be rendered as `some thing` while linking the proper target.

Reply:
Cool, so one step if to remove word references and include correct syntax for class references.  We should be able to deprecate most of these then.

Reply:
Example, I removed `--threadless`.  I realize why you had to add it then to get the integration going.  Should be possible to avoid that entire list of any and py roles.

Reply:
Tracking in https://github.com/abhinavsingh/proxy.py/issues/776